packages/components/src/VirtualizedList/RowLarge/RowLarge.scss

```diff
@@ -24,6 +24,9 @@ $tc-list-large-field-value-color: $dove-gray !default;
 			flex-grow: 1;
 			flex-shrink: 1;
 		}
+		:global(.cell-title-actions) {
```

Comment:
I still have a bad behavior with this one
![image](https://user-images.githubusercontent.com/18534166/46803244-515ad400-cd68-11e8-9e1a-dbbf24615de1.png)